client/my-sites/email/gsuite-purchase-cta/index.jsx

```diff
@@ -178,15 +120,13 @@ GSuitePurchaseCta.propTypes = {
 	currencyCode: PropTypes.string,
 	domainName: PropTypes.string.isRequired,
 	gsuiteBasicCost: PropTypes.number,
-	gsuiteBusinessCost: PropTypes.number,
```

Comment:
We should remove that prop from [the unit test](https://github.com/Automattic/wp-calypso/blob/5e8a3ae9d73682d827d240bd57b45f3c4b5bc246/client/my-sites/email/gsuite-purchase-cta/test/index.js#L28) as well.